app.rb

```diff
@@ -3,6 +3,8 @@
 require "sinatra/base"
```

Comment:
Can we replace this with a Bundler require so we don't need to load these individually? https://bundler.io/guides/bundler_setup.html#setting-up-your-application-to-use-bundler